src/app/api/verifyPin/verifyPin.api.ts

```diff
@@ -0,0 +1,33 @@
+import { Client } from "@notionhq/client";
+const notionSecret = process.env.NOTION_SECRET || undefined;
+const masterPin = process.env.MASTER_PIN;
+const notion = new Client({ auth: notionSecret });
+export async function verifyCaptainPin(captainId: string, Pin: string) {
+  try {
+    const response: any = await notion.pages.retrieve({
+      page_id: captainId,
+    });
+    const captainPin = (response.properties["$Pin"].rich_text[0].plain_text).replace(/\n/g, "")
```

Comment:
You can not hardcode the `$Pin` here , this is something the client needs to say (body = {propertyName: "$Pin", value: "123123"

Reply:
@abelbordonado what do you mean here?